activerecord/test/cases/adapters/postgresql/quoting_test.rb

```diff
@@ -29,6 +29,18 @@ def test_quote_float_infinity
           assert_equal "'Infinity'", @conn.quote(infinity)
         end
 
+        def test_cast_bound_integer
```

Comment:
Are these tests describing what they're actually testing?

Reply:
Good catch, the names should be `test_quote_blah_blah`, I guess?

Reply:
They're also testing behaviour that's inherited unchanged from the abstract adapter, I think? Is this not covered elsewhere?